api-reference/_hidden/dxMenuItem/linkAttr.md

```diff
@@ -5,7 +5,123 @@ default: {}
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies link attributes for the [url](/Documentation/ApiReference/UI_Components/dxMenu/Configuration/items/#url) option.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+You can assign all attributes described in the <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes" target="_blank">Attributes</a> article except for the `href` attribute. Use the [url](/Documentation/ApiReference/UI_Components/dxMenu/Configuration/items/#url) property to specify a link instead.
```

Comment:
We don't recommend assign all attributes. Global attributes is not included in this list, except id/class and aria-*